src/snap/invokeSnap.ts

```diff
@@ -0,0 +1,31 @@
+import { Page } from "puppeteer";
+import { SnapInstallationParamNames } from "../types";
+import { flaskOnly } from "./utils";
+
+export async function invokeSnap(
+  page: Page,
+  snapId: string,
+  method: string,
+  params: Record<SnapInstallationParamNames, any> = {}
+): Promise<Partial<any>> {
```

Comment:
The developer should know the return type
```suggestion
export async function invokeSnap<R = unknown>(
  page: Page,
  snapId: string,
  method: string,
  params: unknown[] | Record<string, unknown> = {}
): Promise<R> {
```

Reply:
@BeroBurny Thank you for the review! I have a question about how to fix when I pass params to the page.evaluate() method, if I will do it this way I have an error:
```
Type 'Record<string, unknown> | unknown[]' is not assignable to type 'Serializable'.
```
So I think one of the solutions is to tell explicit expected parameters like version, description, prompt, and textArea. Or what you can suggest?